### PR TITLE
Fix #1603: Schema diagram how to

### LINKED ISSUE #1603: Schema diagram how to
Add steps to generate schema diagram to `Developer-How-To-Start.md`

### PR DESCRIPTION
**Pull request by jnpsk**
You can run the shell code snippets from inside IDEA. Following the tutorial should result in generating the diagram with the right tables.

### CODE REVIEW

File: docs-private/Developer-How-To-Start.md [hunk 1]
```diff
@@ -46,6 +46,56 @@ liquibase --changeLogFile=./docs/db/changelog/changesets/powerauth-java-server/d
...
 
+### Schema Diagram
+
+To generate diagram of the database schema, use [SchemaCrawler](https://www.schemacrawler.com/) tool. Unfortunately,
+the SchemaCrawler cannot be installed via a package manager for MacOS. You can either use the [docker image](https://www.schemacrawler.com/docker-image.html)
+or get the tool from their [releases page](https://github.com/schemacrawler/SchemaCrawler/releases). Note, that
+the SchemaCrawler diagram generation depends on `graphviz`, which is available via `brew` package manager.
+
+To install graphviz and download SchemaCrawler run:
+```shell
```
Comment (banterCZ) [inline docs-private/Developer-How-To-Start.md:57]: I suggest using more space.

```suggestion
To install graphviz and download SchemaCrawler run:

```shell
```

File: docs-private/Developer-How-To-Start.md [hunk 2]
```diff
@@ -46,6 +46,56 @@ liquibase --changeLogFile=./docs/db/changelog/changesets/powerauth-java-server/d
...
+brew install graphviz && \
+curl -Lo schemacrawler.zip "https://github.com/schemacrawler/SchemaCrawler/releases/download/v${SC_VERSION}/schemacrawler-${SC_VERSION}-distribution.zip" && \
+unzip schemacrawler.zip && \
+rm schemacrawler.zip && \
+mv "schemacrawler-${SC_VERSION}-distribution" schema_crawler
+```
+
+There are two config files for the SchemaCrawler, which should be modified.
+To modify the configuration, run:
+```shell
```
Comment (banterCZ) [inline docs-private/Developer-How-To-Start.md:68]:
```suggestion
To modify the configuration, run:

```shell
```

**Review by banterCZ — APPROVED** [review]
What a great improvement, thanks a lot!

**Review by romanstrobl — APPROVED** [review]
Nice!

**Review by banterCZ — APPROVED** [review]

**Pull request merged**